backend/common/fbs/NetEncoding/Column.py

```diff
@@ -4,8 +4,9 @@
 
 import flatbuffers
 
+
```

Comment:
FYI only:   these files in `NetEncoding` are machine-generated.  While there is no harm in linting / prettying them, there is also no benefit, and we'll lose the fixes as soon as we regenerate the files (not commonly done).